### PR TITLE
Integrate plutus-example cabal.project into project cabal.project

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: nix/haskell.nix [hunk 1]
```diff
@@ -27,6 +27,16 @@
...
   ''
+  # Disable plutus-example on Windows because Windows builds are cross-compiled which
+  # does not work for plugins that are used by plutus-example.
+  + lib.optionalString stdenv.hostPlatform.isWindows ''
+    package plutus-example
+      buildable: False
+    package plutus-ledger
+      buildable: False
+    package plutus-tx-plugin
+      buildable: False
```
Comment (newhoggy) [inline nix/haskell.nix:38]: Make these non-buildable.

File: nix/haskell.nix [hunk 2]
```diff
@@ -62,6 +72,15 @@ let
...
         ];
       })
+      {
+        # Tell `release-lib` what to exclude these from windows builds
+        packages.plutus-example.components.library.platforms = with lib.platforms; [ linux darwin ];
+        packages.plutus-example.components.exes.plutus-example.platforms = with lib.platforms; [ linux darwin ];
+        packages.plutus-ledger.components.library.platforms = with lib.platforms; [ linux darwin ];
+        packages.plutus-ledger.components.exes.plutus-example.platforms = with lib.platforms; [ linux darwin ];
+        packages.plutus-tx-plugin.components.library.platforms = with lib.platforms; [ linux darwin ];
+        packages.plutus-tx-plugin.components.exes.plutus-example.platforms = with lib.platforms; [ linux darwin ];
```
Comment (newhoggy) [inline nix/haskell.nix:82]: Skip this packages on Windows.

**Review by MarcFontaine — APPROVED** [review]
LGTM

**Review by erikd — APPROVED** [review]

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**